service/src/main/java/com/ge/predix/acs/audit/PredixEventMapper.java

```diff
@@ -0,0 +1,65 @@
+package com.ge.predix.acs.audit;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ge.predix.audit.AuditEvent;
+import com.ge.predix.audit.sdk.message.AuditEnums.CategoryType;
+import com.ge.predix.audit.sdk.message.AuditEnums.Classifier;
+import com.ge.predix.audit.sdk.message.AuditEnums.EventType;
+import com.ge.predix.audit.sdk.message.AuditEnums.PublisherType;
+import com.ge.predix.audit.sdk.message.AuditEventV2;
+import com.ge.predix.audit.sdk.message.AuditEventV2.AuditEventV2Builder;
+
+@Component
+@Profile("predix")
+public class PredixEventMapper {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PredixEventProcessor.class);
+
+    private ObjectMapper objectMapper;
+
+    public PredixEventMapper() {
+        this.objectMapper = new ObjectMapper();
+    }
+
+    public AuditEventV2 map(final AuditEvent auditEvent) {
+        Map<String, String> auditPayload = new HashMap<>();
+        auditPayload.put("httpMethod", auditEvent.getMethod());
```

Comment:
May next time when we commit we can make hardcoded string to private final String members